<a href="https://colab.research.google.com/github/ahmadluay9/h8dsft/blob/master/h8dsft_P0W3_Ahmad_Luay_Adnani.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#GRADED CHALLENGE 3

Ahmad Luay Adnani

FTDS-018-RMT



---




# Background

Kamu adalah seorang data analis di The Look yang merupakan salah satu platform e-commerce terbesar di planet Mars. Kamu diminta untuk membuat laporan evaluasi aktivitas penjualan di platform tersebut.

Untuk mempermudah pekerjaan kamu supaya terarah, kamu harus menentukan problem statement dengan success criteria berdasarkan SMART. Namun tantangannya, kamu tentukan problem statement berdasarkan penjabaran-penjabaran analisis dalam bentuk persoalan yang harus kamu jawab menggunakan Query SQL. (SMART akan berdasarkan dari poin-poin penjabaran).

---

# Problem Statements

Analisis di perusahaan ini dilakukan dengan metode SMART. Dengan menggunakan metode SMART kita dapat menentukan tujuan, pencapaian, dan menyusun perencanaan untuk masa depan. [Link](https://glints.com/id/lowongan/metode-smart-adalah/#:~:text=SMART%20adalah%20singkatan%20dari%20lima,menentukan%20tujuan%20dari%20sebuah%20proyek.) 

Berikut adalah SMART model yang digunakan pada analisis ini.
- **Spesific** : Berapa banyak jumlah transakasi yang berstatus `complete` ?
- **Measurable** : Kategori produk apa yang paling banyak dibeli dan paling banyak mendapatkan pendapatan ?
- **Achievable** : Apakah pada akhir Q3 tahun 2022 bisa memperoleh keuntungan ?
- **Relevant** : Berapa banyak user yang setia untuk tetap menggunakan platform The Look di akhir Q3 tahun 2022?
- **Time-Bound**: Apakah di Q4 tahun 2022 The Look akan semakin populer ? 

# Assignment Problem

**Poin penjabaran:**

1. Berapa jumlah transaksi yang berstatus Complete tiap bulan selama Q1 sampai Q3 di tahun 2022? Insight apa yang bisa kamu berikan?
2. Berikan informasi total penjualan (dalam USD) tiap bulan selama Q1 sampai Q3 di tahun 2022! (Hanya yang transaksi berstatus Complete. Apa informasi yang bisa kamu sampaikan?
3. Berapa user yang melakukan transaksi berstatus Complete di tiap bulan dari Q1 sampai Q3 2022? Apa kesimpulanmu?
4. Kategori produk apa saja yang paling banyak dibeli (staus transaksi: Complete) di tiap bulannya selama Q1 sampai Q3 tahun 2022? Berikan insight!
5. Kategori produk apa saja yang paling banyak mendapatkan pendapatan (staus transaksi: Complete) di tiap bulannya selama Q1 sampai Q3 tahun 2022? Berikan insight!
6. Dibandingkan dengan jumlah transaksi dan total penjualan, mana yang paling berkaitan dengan jumlah user yang melakukan transaksi? Apa analisis yang dapat kamu jelaskan? (Hint: Kamu bisa menggunakan korelasi)

**POIN ANALISIS**

1. Berikan kesimpulan dari laporan/informasi yang dibuat berdasarkan problem statement/poin penjabaran menggunakan bahasa awam! (boleh dalam beberapa paragraf)

**POIN PERTANYAAN**

1. Apakah problem statement yang kamu definisikan di awal dapat terukur ketercapaiannya? berikan pendapatmu!
2. Berdasarkan hasil analisis yang sudah kamu lakukan dari 6 penjabaran di atas, jika CEO perusahaanmu ingin menargetkan pendapatan di awal kuartal 4 harus mencapai $250000, apakah masuk akal?
3. CEO kamu menargetkan di kuartal 4 ada investor yang dapat menyuntikan dana ke perusahaanmu, dimana investor akan melihat GMV selama 3 kuartal terakhir serta prospek kedepan minimal di kuartal 4 akan seperti apa. Berikan informasi kepada CEO mu berdasarkan trend transaksi, jumlah user yang bertransaksi, dan GMV 3 kuartal terakhir, apakah perusahaanmu layak atau tidak mendapatkan investor baru!

---

# Data Preparation

## Import Library

Library yang digunakan berasal dari `google.cloud`,`google.colab` dan `pandas`. Langkah-langkah untuk import library sebagai berikut :

In [1]:
#import libraries
from google.cloud import bigquery
from google.colab import auth
auth.authenticate_user()
print('Authenticated')

project_id = "firm-crossing-376307" #GUNAKAN GCP PROJECT-ID KALIAN MASING-MASING
client = bigquery.Client(project=project_id)
import pandas as pd
print("Versi pandas : ",pd.__version__)

Authenticated
Versi pandas :  1.3.5


---

## Reviewing Data in Q1 - Q3 2022 

### Transaction Status

#### Overview

Tabel yang digunakan pada query ini adalah tabel `orders` dikarekan didalam tabel tersebut terdapat kolom `status` transaksi dari semua item. Kolom `order_id` pada tabel `orders` digunakan untuk menghitung jumlah transaksi untuk semua jenis status. Dari data diketahui terdapat beberapa status transaksi yaitu `complete`, `cancelled`, `processing`, `shipped` dan `returned`.

Berikut adalah jumlah transaksi dari penjualan dan yang statusnya `complete` tiap bulannya selama Q1-Q3 tahun 2022 :

In [86]:
# Q1-q3 2022 sales yang status `complete`
transaction_status = client.query('''
SELECT 
  EXTRACT(year FROM created_at) AS year, # kolom `year` dari kolom `created_at` table `orders`
  EXTRACT(month FROM created_at) AS month, # kolom `month` dari kolom `created_at` table `orders`
  COUNT(CASE WHEN status='Complete' THEN order_id ELSE NULL END) AS status_complete # kolom `status_complete` dari jumlah data di kolom `status` table `orders`
FROM bigquery-public-data.thelook_ecommerce.orders # FROM table `orders`
WHERE created_at BETWEEN "2022-01-01" AND "2022-09-30" # WHERE dari kolom `created_at` table `orders` kondisi tahun 2022
GROUP BY year,month # di group agar bisa di run
ORDER BY year,month ASC; # di urutkan menurut bulannya
''').to_dataframe()

transaction_status

year  month  status_complete
0  2022      1              855
1  2022      2              815
2  2022      3              929
3  2022      4              975
4  2022      5             1049
5  2022      6             1160
6  2022      7             1205
7  2022      8             1302
8  2022      9             1366

Tabel diatas memperlihatkan jumlah semua status transaksi dan yang statusnya `complete` setiap bulannya selama Q1 sampai Q3 tahun 2022. Dapat dilihat setiap bulannya jumlah transaksi cenderung mengalami kenaikan. Persentase jumlah transaksi yang berstatus `complete` setiap bulannya selama Q1 sampai Q3 tahun 2022 sekitar 25% dari semua status transaksi.

#### Transaction Status Q1

Berikut adalah jumlah transaksi dari semua jenis status penjualan setiap bulannya pada Q1 (Januari - Maret) 2022 :

In [99]:
# Q1, January-March 2022 dengan semua jenis status penjualan
status_q1 = client.query('''
SELECT
  EXTRACT(year FROM created_at) AS year, # kolom `year` dari kolom `created_at` table `orders`
  EXTRACT(month FROM created_at) AS month, # kolom `month` dari kolom `created_at` table `orders`
  COUNT(CASE WHEN status="Complete" THEN order_id ELSE NULL END) AS complete, # kolom `complete` dari jumlah data berstatus `complete`  di kolom `status` table `orders`
FROM `bigquery-public-data.thelook_ecommerce.orders` # FROM table `orders`
WHERE created_at BETWEEN "2022-01-01" AND "2022-04-01"  # q1 januari - maret
GROUP BY year,month  # di group agar bisa di run
ORDER BY year,month ASC # di urutkan berdasarkan bulannya
''').to_dataframe()

status_q1

year  month  complete
0  2022      1       855
1  2022      2       815
2  2022      3       929

Tabel diatas memperlihatkan jumlah transaksi yang berstatus penjualan `complete` pada **Q1 (Januari - Maret) 2022**. Dari tabel diatas didapatkan informasi :
- Di bulan **Januari 2022** terdapat 3.352 transaksi dengan jumlah transaksi yang berstatus `complete` sebanyak **855** transaksi.
- Di bulan **Februari 2022** jumlah transanksi yang berstatus `complete` mengalami penurunan sebanyak 40 transaksi menjadi **815** transaksi. 
- Di Bulan **Maret 2022** jumlah transanksi yang berstatus `complete` mengalami kenaikan sebanyak 114 transaksi menjadi **929** transaksi.

#### Transaction Status Q2

Berikut adalah jumlah transaksi dari semua jenis status penjualan setiap bulannya pada Q2 (April - Juni) 2022 :

In [100]:
# Q2, April-June 2022 dengan semua jenis status penjualan
status_q2 = client.query('''
SELECT
  EXTRACT(year FROM created_at) AS year, # kolom `year` dari kolom `created_at` table `orders`
  EXTRACT(month FROM created_at) AS month, # kolom `month` dari kolom `created_at` table `orders`
  COUNT(CASE WHEN status="Complete" THEN order_id ELSE NULL END) AS complete, # kolom `complete` dari jumlah data berstatus `complete`  di kolom `status` table `orders`
FROM `bigquery-public-data.thelook_ecommerce.orders` # FROM table `orders`
WHERE created_at BETWEEN "2022-04-01" AND "2022-07-01"  # q2 april - juni
GROUP BY year,month  # di group agar bisa di run
ORDER BY year,month ASC # di urutkan berdasarkan bulannya
''').to_dataframe()

status_q2

year  month  complete
0  2022      4       975
1  2022      5      1049
2  2022      6      1160

Tabel diatas memperlihatkan jumlah transaksi yang berstatus penjualan `complete` pada **Q2 (April - Juni) 2022**. Dari tabel diatas didapatkan informasi : 
- Di bulan **April 2022** jumlah transaksi yang berstatus `complete` mengalami kenaikan sebanyak 46 transaksi menjadi **975** transaksi.
- Di bulan **Mei 2022** jumlah transaksi yang berstatus `complete` mengalami kenaikan sebanyak 74 transaksi menjadi **1049** transaksi. 
- Di Bulan **Juni 2022** jumlah transaksi yang berstatus `complete` mengalami kenaikan sebanyak 111 transaksi menjadi **1160** transaksi.

#### Transaction Status Q3



Berikut adalah jumlah transaksi dari semua jenis status penjualan setiap bulannya pada Q3 (Juli - September) 2022 :

In [101]:
# Q3, July - September 2022 dengan semua jenis status penjualan
status_q3 = client.query('''
SELECT
  EXTRACT(year FROM created_at) AS year, # kolom `year` dari kolom `created_at` table `orders`
  EXTRACT(month FROM created_at) AS month, # kolom `month` dari kolom `created_at` table `orders`
  COUNT(CASE WHEN status="Complete" THEN order_id ELSE NULL END) AS complete, # kolom `complete` dari jumlah data berstatus `complete`  di kolom `status` table `orders`
FROM `bigquery-public-data.thelook_ecommerce.orders` # FROM table `orders`
WHERE created_at BETWEEN "2022-07-01" AND "2022-09-30"  # q3 juli - september
GROUP BY year,month  # di group agar bisa di run
ORDER BY year,month ASC # di urutkan berdasarkan bulannya
''').to_dataframe()

status_q3

year  month  complete
0  2022      7      1205
1  2022      8      1302
2  2022      9      1366

Tabel diatas memperlihatkan jumlah transaksi yang berstatus penjualan `complete` pada **Q3 (Juli - September) 2022**. Dari tabel diatas didapatkan informasi :
- Di bulan **Juli 2022** jumlah transaksi yang berstatus `complete` mengalami kenaikan sebanyak 45 transaksi menjadi **1205** transaksi.
- Di bulan **Agustus 2022** jumlah transaksi yang berstatus `complete` mengalami kenaikan sebanyak 97 transaksi menjadi **1302** transaksi. 
- Di Bulan **September 2022** jumlah transaksi yang berstatus `complete` mengalami kenaikan sebanyak 64 transaksi menjadi **1366** transaksi.

### Total Revenue

#### Overview

*Revenue* adalah jumlah pendapatan yang dihasilkan dari penjualan barang dan jasa yang terkait dengan operasi utama bisnis. 

Tabel yang digunakan untuk query ini yaitu tabel `order_items`. Di tabel `order_items` terdapat kolom `sale_price` yang menunjukkan total penjualan. 
 Berikut adalah total penjualan (dalam USD) untuk transaksi berstatus `complete` setiap bulannya selama Q1-Q3 tahun 2022 :

In [88]:
# Total Revenue of Q1~Q3 with status Complete
revenue_all = client.query('''
SELECT Year, Month,
  round(Total_Sales) AS Total_Sales
FROM ( SELECT 
       EXTRACT(year from items.created_at) as Year, 
       EXTRACT(month from items.created_at) as Month, 
       SUM(items.sale_price) as Total_Sales
FROM `bigquery-public-data.thelook_ecommerce.order_items` AS items
WHERE items.created_at BETWEEN '2022-01-01' AND '2022-09-30' AND status = 'Complete'
GROUP BY year,month
ORDER BY year,month)
''').to_dataframe()

revenue_all

Year  Month  Total_Sales
0  2022      1      71538.0
1  2022      2      67929.0
2  2022      3      77514.0
3  2022      4      83984.0
4  2022      5      89469.0
5  2022      6     101666.0
6  2022      7     105797.0
7  2022      8     113440.0
8  2022      9     116349.0

Tabel diatas menunjukkan total penjualan (dalam USD) untuk transaksi berstatus `complete` setiap bulan selama Q1 sampai Q3 tahun 2022. Dapat dilihat total penjualan setiap bulannya selama Q1 sampai Q3 tahun 2022 cenderung mengalami kenaikkan. Kenaikan total penjualan terbesar terjadi di bulan Juni 2022 (total penjualan **101.666** USD) yaitu mengalami kenaikan 12.197 USD dari bulan Mei 2022 (total penjualan **89.469** USD).

#### Total Sales Q1

Berikut adalah total pengeluaran dan total penjualan (dalam USD) untuk transaksi berstatus `complete` setiap bulannya pada Q1 (Januari - Maret) tahun 2022 :

In [95]:
# Total Revenue of Q1 with status Complete
revenue_q1 = client.query('''
SELECT Year, Month,
  round(Total_Sales) AS Total_Sales
FROM ( SELECT 
       EXTRACT(year from items.created_at) as Year, 
       EXTRACT(month from items.created_at) as Month, 
       SUM(items.sale_price) as Total_Sales
FROM `bigquery-public-data.thelook_ecommerce.order_items` AS items
WHERE items.created_at BETWEEN '2022-01-01' AND '2022-04-01' AND status = 'Complete'
GROUP BY year,month
ORDER BY year,month)
''').to_dataframe()

revenue_q1

Year  Month  Total_Sales
0  2022      1      71538.0
1  2022      2      67929.0
2  2022      3      77514.0

Tabel diatas memperlihatkan total pengeluaran dan total penjualan pada Q1(Januari - Maret) tahun 2022. Dari tabel diatas didapatkan informasi:
- Total penjualan untuk transaksi berstatus `complete` bulan Januari 2022 sebesar **71.538** USD.
- Total penjualan untuk transaksi berstatus `complete` di bulan Februari 2022 sebesar **67.929** USD, terjadi penurunan sebesar 3.609  USD dari total penjualan bulan sebelumnya. 
- Total penjualan untuk transaksi berstatus `complete` di bulan Maret 2022 sebesar **77.514** USD, terjadi kenaikan sebesar 9.585 USD dari total penjualan bulan sebelumnya.

#### Total Revenue Q2

Berikut adalah total pengeluaran dan total penjualan (dalam USD) untuk transaksi berstatus `complete` setiap bulannya pada Q2 (April - Juni) tahun 2022 :

In [94]:
# Total Revenue of Q2 with status Complete
revenue_q2 = client.query('''
SELECT Year, Month,
  round(Total_Sales) AS Total_Sales
FROM ( SELECT 
       EXTRACT(year from items.created_at) as Year, 
       EXTRACT(month from items.created_at) as Month, 
       SUM(items.sale_price) as Total_Sales
FROM `bigquery-public-data.thelook_ecommerce.order_items` AS items
WHERE items.created_at BETWEEN '2022-04-01' AND '2022-07-01' AND status = 'Complete'
GROUP BY year,month
ORDER BY year,month)
''').to_dataframe()

revenue_q2

Year  Month  Total_Sales
0  2022      4      83984.0
1  2022      5      89469.0
2  2022      6     101666.0

Tabel diatas memperlihatkan total pengeluaran dan total penjualan pada Q2(April - Juni) tahun 2022. Dari tabel diatas didapatkan informasi :  
- Total penjualan untuk transaksi berstatus `complete` bulan April 2022 sebesar **83.984** USD, terjadi kenaikan sebesar 8.802 USD dari total penjualan bulan sebelumnya. 
- Total penjualan untuk transaksi berstatus `complete` di bulan Mei 2022 sebesar **89.469** USD, terjadi kenaikan sebesar 5.485  USD dari total penjualan bulan sebelumnya. 
- Total penjualan untuk transaksi berstatus `complete` di bulan Juni 2022 sebesar **101.666** USD, terjadi kenaikan sebesar 12.197 USD dari total penjualan bulan sebelumnya.

#### Total Revenue Q3


Berikut adalah total pengeluaran dan total penjualan (dalam USD) untuk transaksi berstatus `complete` setiap bulannya pada Q3 (Juli - September) tahun 2022 :

In [96]:
# Total Revenue of Q3 with status Complete
revenue_q3 = client.query('''
SELECT Year, Month,
  round(Total_Sales) AS Total_Sales
FROM ( SELECT 
       EXTRACT(year from items.created_at) as Year, 
       EXTRACT(month from items.created_at) as Month, 
       SUM(items.sale_price) as Total_Sales
FROM `bigquery-public-data.thelook_ecommerce.order_items` AS items
WHERE items.created_at BETWEEN '2022-07-01' AND '2022-09-30' AND status = 'Complete'
GROUP BY year,month
ORDER BY year,month)
''').to_dataframe()

revenue_q3

Year  Month  Total_Sales
0  2022      7     105797.0
1  2022      8     113440.0
2  2022      9     116349.0

Tabel diatas memperlihatkan total pengeluaran dan total penjualan pada Q3(Juli - September) tahun 2022. Dari tabel diatas didapatkan informasi : 
- Total penjualan untuk transaksi berstatus `complete` bulan Juli 2022 sebesar **105.797** USD, terjadi kenaikan sebesar 8.534 USD dari total penjualan bulan sebelumnya. 
- Total penjualan untuk transaksi berstatus `complete` di bulan Agustus 2022 sebesar **113.440** USD, terjadi kenaikan sebesar 7.643  USD dari total penjualan bulan sebelumnya. 
- Total penjualan untuk transaksi berstatus `complete` di bulan September 2022 sebesar **116.349** USD, terjadi kenaikan sebesar 2.909 USD dari total penjualan bulan sebelumnya.

### Number of Users

#### Overview

Tabel yang digunakan untuk query ini adalah tabel `orders`. Di dalam tabel `orders` terdapat kolom `user_id` menunjukkan id dari user yang dapat dijumlahkan untuk mendapatkan jumlah user. Berikut adalah tabel yang memperlihatkan berapa banyak jumlah user(pelanggan) yang melakukan transaksi berstatus `complete` setiap bulan selama Q1 sampai Q3 tahun 2022.

In [91]:
# from table inventory_items and order_items
num_of_users = client.query('''

SELECT
  EXTRACT(year FROM created_at) AS year,
  EXTRACT(month FROM created_at) AS month,
  COUNT(DISTINCT CASE WHEN status = "Complete" THEN user_id ELSE NULL END) AS num_users_complete, # total user status complete
FROM `bigquery-public-data.thelook_ecommerce.orders`
WHERE created_at BETWEEN "2022-01-01" AND "2022-09-30" 
GROUP BY year,month
ORDER BY year,month ASC

''').to_dataframe()
num_of_users

year  month  num_users_complete
0  2022      1                 847
1  2022      2                 810
2  2022      3                 923
3  2022      4                 969
4  2022      5                1044
5  2022      6                1150
6  2022      7                1192
7  2022      8                1292
8  2022      9                1354

Dari tabel diatas didapatkan jumlah user yang melakukan transaksi berstatus `complete` setiap bulan selama Q1 sampai Q3 tahun 2022. Dapat dilihat jumlah user selama Q1 sampai Q3 cenderung mengalami kenaikkan hal ini menunjukkan tidak terjadi **churn** (kehilangan pelanggan).

#### Number of Users Q1

Berikut adalah jumlah user yang melakukan transaksi setiap bulan pada Q1 (Januari - Maret) tahun 2022:

In [92]:
# Number of Users Q1 (Januari - Maret) 2022
num_of_usersq1 = client.query('''

SELECT
  EXTRACT(year FROM created_at) AS year,
  EXTRACT(month FROM created_at) AS month,
  COUNT(DISTINCT CASE WHEN status = "Complete" THEN user_id ELSE NULL END) AS num_users_complete, # total user status complete
FROM `bigquery-public-data.thelook_ecommerce.orders`
WHERE created_at BETWEEN "2022-01-01" AND "2022-04-01" 
GROUP BY year,month
ORDER BY year,month ASC

''').to_dataframe()
num_of_usersq1

year  month  num_users_complete
0  2022      1                 847
1  2022      2                 810
2  2022      3                 923

Tabel diatas memperlihatkan jumlah user yang melakukan transaksi berstatus `complete` setiap bulan pada Q1 (Januari - Maret) tahun 2022. Dari tabel diatas didapatkan informasi :
- Jumlah user yang melakukan transaksi berstatus `complete` pada bulan Januari 2022 sebanyak **847** orang. 
- Jumlah user yang melakukan transaksi berstatus `complete` bulan Februari 2022 sebanyak **810** orang, hal ini menunjukkan terjadi penurunan jumlah user sebanyak 37 orang. 
- Jumlah user yang melakukan transaksi berstatus `complete` bulan Maret 2022 sebanyak **923** orang, hal ini menunjukkan terjadi kenaikan jumlah user sebanyak 113 orang.

#### Number of Users Q2

Berikut adalah jumlah user yang melakukan transaksi setiap bulan pada Q2 (April - Juni) tahun 2022:

In [102]:
# Number of Users Q2 (April - Juni) 2022
num_of_usersq2 = client.query('''

SELECT
  EXTRACT(year FROM created_at) AS year,
  EXTRACT(month FROM created_at) AS month,
  COUNT(DISTINCT CASE WHEN status = "Complete" THEN user_id ELSE NULL END) AS num_users_complete, # total user status complete
FROM `bigquery-public-data.thelook_ecommerce.orders`
WHERE created_at BETWEEN "2022-04-01" AND "2022-07-01" 
GROUP BY year,month
ORDER BY year,month ASC

''').to_dataframe()
num_of_usersq2

year  month  num_users_complete
0  2022      4                 969
1  2022      5                1044
2  2022      6                1150

Tabel diatas memperlihatkan jumlah user yang melakukan transaksi berstatus `complete` setiap bulan pada Q2 (April - Juni) tahun 2022. Dari tabel diatas didapatkan informasi :
- jumlah user yang melakukan transaksi berstatus `complete` pada bulan April 2022 sebanyak **969** orang, hal ini menunjukkan terjadi kenaikan jumlah user sebanyak 46 orang. 
- Jumlah user yang melakukan transaksi berstatus `complete` bulan Mei 2022 sebanyak **1044** orang, hal ini menunjukkan terjadi kenaikan jumlah user sebanyak 75 orang. 
- Jumlah user yang melakukan transaksi berstatus `complete` bulan Juni 2022 sebanyak **1150** orang, hal ini menunjukkan terjadi kenaikan jumlah user sebanyak 106 orang.

#### Number of Users Q3

Berikut adalah jumlah user yang melakukan transaksi setiap bulan pada Q3 (Juli - September) tahun 2022:

In [75]:
# Number of Users Q3 (Juli - September) 2022
num_of_usersq3 = client.query('''

SELECT
  EXTRACT(year FROM created_at) AS year,
  EXTRACT(month FROM created_at) AS month,
  COUNT(DISTINCT CASE WHEN status = "Complete" THEN user_id ELSE NULL END) AS num_users_complete, # total user status complete
FROM `bigquery-public-data.thelook_ecommerce.orders`
WHERE created_at BETWEEN "2022-07-01" AND "2022-09-30" 
GROUP BY year,month
ORDER BY year,month ASC

''').to_dataframe()
num_of_usersq3

year  month  num_users_complete
0  2022      7                1192
1  2022      8                1292
2  2022      9                1354

Tabel diatas memperlihatkan jumlah user yang melakukan transaksi berstatus `complete` setiap bulan pada Q3 (Juli - September) tahun 2022. Dari tabel diatas didapatkan informasi :
- jumlah user yang melakukan transaksi berstatus `complete` pada bulan Juli 2022 sebanyak **1192** orang, hal ini menunjukkan terjadi kenaikan jumlah user sebanyak 40 orang. 
- Jumlah user yang melakukan transaksi berstatus `complete` bulan Agustus 2022 sebanyak **1292** orang, hal ini menunjukkan terjadi kenaikan jumlah user sebanyak 100 orang. 
- Jumlah user yang melakukan transaksi berstatus `complete` bulan September 2022 sebanyak **1354** orang, hal ini menunjukkan terjadi kenaikan jumlah user sebanyak 62 orang.

### Frequently Purchased Products (Quantity) 

#### Overview

Tabel yang digunakan untuk query ini adalah tabel `order_items` dan tabel `inventory_items`. Primary key dari tabel `order_items` dan Foreign key dari tabel `inventory_items` yaitu pada kolom `product_id`. Di tabel `inventory_items` terdapat kolom `product_category` yang digunakan untuk mendapat jumlah kategori produk yang paling banyak dibeli dengan cara diurutkan berdasarkan jumlah yang paling banyak dibeli. Berikut adalah kategori produk yang paling sering dibeli dengan status `complete` setiap bulannya selama Q1 sampai Q3 tahun 2022 :

In [104]:
# Frequently purchased products based on its quantity with rank #1 each month
rank_quantity = client.query('''
SELECT year, month, category, count_category, rank
FROM (SELECT
        year, month, category, count_category,
        RANK() OVER (PARTITION BY month ORDER BY count_category DESC) AS rank # di rank untuk mengetahui yang paling banyak dibeli
      FROM (SELECT DISTINCT
              EXTRACT(year FROM i.created_at) AS year,
              EXTRACT(month FROM i.created_at) AS month,
              ii.product_category AS category, # jenis kategori
              COUNT(ii.product_category) AS count_category, # jumlah kategori
            FROM `bigquery-public-data.thelook_ecommerce.order_items` AS i
            JOIN `bigquery-public-data.thelook_ecommerce.inventory_items` AS ii
            ON i.product_id = ii.product_id # primary key & foreign key
            WHERE i.status = "Complete" AND
                  i.created_at BETWEEN "2022-01-01" AND "2022-09-30"
            GROUP BY year, month, category) count_table_category
            ) count_table_category_rank
WHERE rank = 1 # rank 1 menunjukkan paling banyak dibeli
ORDER BY month 
''').to_dataframe()

rank_quantity

year  month                       category  count_category  rank
0  2022      1                          Jeans            1732     1
1  2022      2  Fashion Hoodies & Sweatshirts            1909     1
2  2022      3                    Tops & Tees            2531     1
3  2022      4                    Tops & Tees            2220     1
4  2022      5                      Intimates            2326     1
5  2022      6  Fashion Hoodies & Sweatshirts            2336     1
6  2022      7                          Jeans            2711     1
7  2022      8                          Jeans            2918     1
8  2022      9                    Tops & Tees            2939     1

Tabel diatas menunjukkan kategori produk yang paling banyak dibeli (status transaksi: `complete`) setiap bulan selama Q1 sampai Q3 tahun 2022. Kategori **Tops & Tees** merupakan kategori produk yang paling banyak dibeli (status transaksi: `complete`) pada Q1 sampai Q3 tahun 2022.

Untuk penjabaran setiap quarter dapat dilihat pada tabel di bawah. 

#### Frequently Purchased Products (Quantity) Q1

Berikut adalah tabel kategori produk yang paling sering dibeli dengan status `complete` pada Q1(Januari - Maret) tahun 2022 :

In [105]:
# frequently purchased product q1 (january - march)
rank_quantity_q1 = client.query('''
SELECT year, month, category, count_category, rank
FROM (SELECT
        year, month, category, count_category,
        RANK() OVER (PARTITION BY month ORDER BY count_category DESC) AS rank # di rank untuk mengetahui yang paling banyak dibeli
      FROM (SELECT DISTINCT
              EXTRACT(year FROM i.created_at) AS year,
              EXTRACT(month FROM i.created_at) AS month,
              ii.product_category AS category, # jenis kategori
              COUNT(ii.product_category) AS count_category, # jumlah kategori
            FROM `bigquery-public-data.thelook_ecommerce.order_items` AS i
            JOIN `bigquery-public-data.thelook_ecommerce.inventory_items` AS ii
            ON i.product_id = ii.product_id # primary key & foreign key
            WHERE i.status = "Complete" AND
                  i.created_at BETWEEN "2022-01-01" AND "2022-04-01"
            GROUP BY year, month, category) count_table_category
            ) count_table_category_rank
WHERE rank = 1 # rank 1 menunjukkan paling banyak dibeli
ORDER BY month 
''').to_dataframe()
rank_quantity_q1

year  month                       category  count_category  rank
0  2022      1                          Jeans            1732     1
1  2022      2  Fashion Hoodies & Sweatshirts            1909     1
2  2022      3                    Tops & Tees            2531     1

Dari tabel diatas didapatkan informasi : 
- Kategori produk yang paling banyak di beli di bulan Januari 2022 yaitu **Jeans** sebanyak **1.732** unit.
- Kategori produk yang paling banyak di beli di bulan Februari 2022 yaitu **Fashion Hoodies & Sweatshirts** sebanyak **1.909** unit.
- Kategori produk yang paling banyak di beli di bulan Maret 2022 yaitu **Tops & Tees** sebanyak **2.531** unit.

#### Frequently Purchased Products (Quantity) Q2

Berikut adalah tabel kategori produk yang paling sering dibeli dengan status `complete` pada Q2(April - Juni) tahun 2022 :

In [106]:
# frequently purchased product q2 (april - june )
rank_quantity_q2 = client.query('''
SELECT year, month, category, count_category, rank
FROM (SELECT
        year, month, category, count_category,
        RANK() OVER (PARTITION BY month ORDER BY count_category DESC) AS rank # di rank untuk mengetahui yang paling banyak dibeli
      FROM (SELECT DISTINCT
              EXTRACT(year FROM i.created_at) AS year,
              EXTRACT(month FROM i.created_at) AS month,
              ii.product_category AS category, # jenis kategori
              COUNT(ii.product_category) AS count_category, # jumlah kategori
            FROM `bigquery-public-data.thelook_ecommerce.order_items` AS i
            JOIN `bigquery-public-data.thelook_ecommerce.inventory_items` AS ii
            ON i.product_id = ii.product_id # primary key & foreign key
            WHERE i.status = "Complete" AND
                  i.created_at BETWEEN "2022-04-01" AND "2022-07-01"
            GROUP BY year, month, category) count_table_category
            ) count_table_category_rank
WHERE rank = 1 # rank 1 menunjukkan paling banyak dibeli
ORDER BY month 
''').to_dataframe()
rank_quantity_q2

year  month                       category  count_category  rank
0  2022      4                    Tops & Tees            2220     1
1  2022      5                      Intimates            2326     1
2  2022      6  Fashion Hoodies & Sweatshirts            2336     1

Dari tabel diatas didapatkan informasi : 
- Kategori produk yang paling banyak di beli di bulan April 2022 yaitu **Tops & Tees** sebanyak **2.220** unit.
- Kategori produk yang paling banyak di beli di bulan Mei 2022 yaitu **Intimates** sebanyak **2.326** unit.
- Kategori produk yang paling banyak di beli di bulan Juni 2022 yaitu **Fashion Hoodies & Sweatshirts** sebanyak **2.336** unit.


#### Frequently Purchased Products (Quantity) Q3

Berikut adalah tabel kategori produk yang paling sering dibeli dengan status `complete` pada Q3(Juli - September) tahun 2022 :

In [107]:
# frequently purchased product q3 (july - september)
rank_quantity_q3 = client.query('''
SELECT year, month, category, count_category, rank
FROM (SELECT
        year, month, category, count_category,
        RANK() OVER (PARTITION BY month ORDER BY count_category DESC) AS rank # di rank untuk mengetahui yang paling banyak dibeli
      FROM (SELECT DISTINCT
              EXTRACT(year FROM i.created_at) AS year,
              EXTRACT(month FROM i.created_at) AS month,
              ii.product_category AS category, # jenis kategori
              COUNT(ii.product_category) AS count_category, # jumlah kategori
            FROM `bigquery-public-data.thelook_ecommerce.order_items` AS i
            JOIN `bigquery-public-data.thelook_ecommerce.inventory_items` AS ii
            ON i.product_id = ii.product_id # primary key & foreign key
            WHERE i.status = "Complete" AND
                  i.created_at BETWEEN "2022-07-01" AND "2022-09-30"
            GROUP BY year, month, category) count_table_category
            ) count_table_category_rank
WHERE rank = 1 # rank 1 menunjukkan paling banyak dibeli
ORDER BY month 
''').to_dataframe()
rank_quantity_q3

year  month     category  count_category  rank
0  2022      7        Jeans            2711     1
1  2022      8        Jeans            2918     1
2  2022      9  Tops & Tees            2939     1

Dari tabel diatas didapatkan informasi : 
- Kategori produk yang paling banyak di beli di bulan Juli 2022 yaitu **Jeans** sebanyak **2.711** unit.
- Kategori produk yang paling banyak di beli di bulan Agustus 2022 yaitu **Intimates** sebanyak **2.918** unit.
- Kategori produk yang paling banyak di beli di bulan September 2022 yaitu **Tops & Tees** sebanyak **2.939** unit.

### The Highest Revenue of Frequently Purchased Products

#### Overview

Tabel yang digunakan untuk query ini adalah tabel `order_items` dan tabel `inventory_items`. Primary key dari tabel `order_items` dan Foreign key dari tabel `inventory_items` yaitu pada kolom `product_id`. Di tabel `order_items` terdapat kolom `sale_price` yang dapat menunjukkan total penjualan suatu kategori produk. Di tabel `inventory_items` terdapat kolom `product_category` yang digunakan untuk mengetahui kategori produk yang paling banyak mendapatkan pendapatan. Berikut adalah kategori produk yang paling banyak mendapatkan pendapatan dengan status `complete` setiap bulannya selama Q1 sampai Q3 tahun 2022 :

In [108]:
# Revenue each product category rank #1 each month
rank_revenue = client.query('''
SELECT year, month, category,
       ROUND(revenue_category,2) as revenue,
       rank
FROM (SELECT
        year,
        month,
        category,
        revenue_category,
        RANK() OVER (PARTITION BY month ORDER BY revenue_category DESC) AS rank # rank untuk mengetahui yang paling banyak mendapatkan pendapatan
      FROM (SELECT DISTINCT
              EXTRACT(year FROM i.created_at) AS year,
              EXTRACT(month FROM i.created_at) AS month,
              ii.product_category AS category, # jenis kategori
              SUM(i.sale_price) AS revenue_category, # total pendapatan
            FROM `bigquery-public-data.thelook_ecommerce.order_items` AS i
            LEFT JOIN `bigquery-public-data.thelook_ecommerce.inventory_items` AS ii
            ON i.product_id = ii.product_id # primary  key and foreign key
            WHERE i.status = "Complete" AND
                  i.created_at BETWEEN "2022-01-01" AND "2022-09-30"
            GROUP BY year, month, category) revenue_table_category
            ) revenue_table_category_rank
WHERE rank = 1 # rank 1 menunjukkan paling banyak mendapatkan pendapatan
ORDER BY month 
''').to_dataframe()
rank_revenue

year  month           category    revenue  rank
0  2022      1              Jeans  180011.98     1
1  2022      2              Jeans  180244.74     1
2  2022      3              Jeans  188664.93     1
3  2022      4  Outerwear & Coats  180798.84     1
4  2022      5              Jeans  207226.65     1
5  2022      6  Outerwear & Coats  284904.50     1
6  2022      7              Jeans  291556.34     1
7  2022      8              Jeans  317618.18     1
8  2022      9  Outerwear & Coats  307363.79     1

Tabel diatas menunjukkan kategori produk yang paling banyak mendapatkan pendapatan (status transaksi: `complete`) setiap bulan selama Q1 sampai Q3 tahun 2022. Dari tabel diatas kategori **Jeans** merupakan kategori yang paling banyak mendapatkan pendapatan (status transaksi: `complete`) pada Q1 sampai Q3 tahun 2022.

Untuk penjabaran setiap quarter nya dapat dilihat pada tabel di bawah.

#### The Highest Revenue of Frequently Purchased Products Q1

Berikut adalah tabel kategori produk yang paling banyak mendapatkan pendapatan dengan status `complete` pada Q1(Januari - Maret) tahun 2022 :

In [109]:
# kategori produk yang paling banyak mendapatkan pendapatan q1
rank_revenue_q1 = client.query('''
SELECT year, month, category,
       ROUND(revenue_category,2) as revenue,
       rank
FROM (SELECT
        year,
        month,
        category,
        revenue_category,
        RANK() OVER (PARTITION BY month ORDER BY revenue_category DESC) AS rank # rank untuk mengetahui yang paling banyak mendapatkan pendapatan
      FROM (SELECT DISTINCT
              EXTRACT(year FROM i.created_at) AS year,
              EXTRACT(month FROM i.created_at) AS month,
              ii.product_category AS category, # jenis kategori
              SUM(i.sale_price) AS revenue_category, # total pendapatan
            FROM `bigquery-public-data.thelook_ecommerce.order_items` AS i
            LEFT JOIN `bigquery-public-data.thelook_ecommerce.inventory_items` AS ii
            ON i.product_id = ii.product_id # primary  key and foreign key
            WHERE i.status = "Complete" AND
                  i.created_at BETWEEN "2022-01-01" AND "2022-04-01"
            GROUP BY year, month, category) revenue_table_category
            ) revenue_table_category_rank
WHERE rank = 1 # rank 1 menunjukkan paling banyak mendapatkan pendapatan
ORDER BY month 
''').to_dataframe()
rank_revenue_q1

year  month category    revenue  rank
0  2022      1    Jeans  180011.98     1
1  2022      2    Jeans  180244.74     1
2  2022      3    Jeans  188664.93     1

Dari tabel diatas didapatkan informasi :
- Kategori produk dengan pendapatan (status transaksi: complete) terbanyak di bulan Januari 2022 adalah **jeans** dengan total pendapatan sebesar **180.011** USD
- Kategori produk dengan pendapatan (status transaksi: complete) terbanyak di bulan Februari 2022 adalah **jeans** dengan total pendapatan sebesar **180.244** USD
- Kategori produk dengan pendapatan (status transaksi: complete) terbanyak di bulan Maret 2022 adalah **jeans** dengan total pendapatan sebesar **188.664** USD

#### The Highest Revenue of Frequently Purchased Products Q2

Berikut adalah tabel kategori produk yang paling banyak mendapatkan pendapatan dengan status `complete` pada Q2(April - Juni) tahun 2022 :

In [110]:
# kategori produk yang paling banyak mendapatkan pendapatan q2
rank_revenue_q2 = client.query('''
SELECT year, month, category,
       ROUND(revenue_category,2) as revenue,
       rank
FROM (SELECT
        year,
        month,
        category,
        revenue_category,
        RANK() OVER (PARTITION BY month ORDER BY revenue_category DESC) AS rank # rank untuk mengetahui yang paling banyak mendapatkan pendapatan
      FROM (SELECT DISTINCT
              EXTRACT(year FROM i.created_at) AS year,
              EXTRACT(month FROM i.created_at) AS month,
              ii.product_category AS category, # jenis kategori
              SUM(i.sale_price) AS revenue_category, # total pendapatan
            FROM `bigquery-public-data.thelook_ecommerce.order_items` AS i
            LEFT JOIN `bigquery-public-data.thelook_ecommerce.inventory_items` AS ii
            ON i.product_id = ii.product_id # primary  key and foreign key
            WHERE i.status = "Complete" AND
                  i.created_at BETWEEN "2022-04-01" AND "2022-07-01"
            GROUP BY year, month, category) revenue_table_category
            ) revenue_table_category_rank
WHERE rank = 1 # rank 1 menunjukkan paling banyak mendapatkan pendapatan
ORDER BY month 
''').to_dataframe()
rank_revenue_q2

year  month           category    revenue  rank
0  2022      4  Outerwear & Coats  180798.84     1
1  2022      5              Jeans  207226.65     1
2  2022      6  Outerwear & Coats  284904.50     1

Dari tabel diatas didapatkan informasi :
- Kategori produk dengan pendapatan (status transaksi: complete) terbanyak di bulan April 2022 adalah **Outerwear & Coats** dengan total pendapatan sebesar **180.798** USD
- Kategori produk dengan pendapatan (status transaksi: complete) terbanyak di bulan Mei 2022 adalah **jeans** dengan total pendapatan sebesar **207.226** USD
- Kategori produk dengan pendapatan (status transaksi: complete) terbanyak di bulan Juni 2022 adalah **Outerwear & Coats** dengan total pendapatan sebesar **284.904** USD

#### The Highest Revenue of Frequently Purchased Products Q3

Berikut adalah tabel kategori produk yang paling banyak mendapatkan pendapatan dengan status `complete` pada Q3(Juli - September) tahun 2022 :

In [111]:
# kategori produk yang paling banyak mendapatkan pendapatan q3
rank_revenue_q3 = client.query('''
SELECT year, month, category,
       ROUND(revenue_category,2) as revenue,
       rank
FROM (SELECT
        year,
        month,
        category,
        revenue_category,
        RANK() OVER (PARTITION BY month ORDER BY revenue_category DESC) AS rank # rank untuk mengetahui yang paling banyak mendapatkan pendapatan
      FROM (SELECT DISTINCT
              EXTRACT(year FROM i.created_at) AS year,
              EXTRACT(month FROM i.created_at) AS month,
              ii.product_category AS category, # jenis kategori
              SUM(i.sale_price) AS revenue_category, # total pendapatan
            FROM `bigquery-public-data.thelook_ecommerce.order_items` AS i
            LEFT JOIN `bigquery-public-data.thelook_ecommerce.inventory_items` AS ii
            ON i.product_id = ii.product_id # primary  key and foreign key
            WHERE i.status = "Complete" AND
                  i.created_at BETWEEN "2022-07-01" AND "2022-09-30"
            GROUP BY year, month, category) revenue_table_category
            ) revenue_table_category_rank
WHERE rank = 1 # rank 1 menunjukkan paling banyak mendapatkan pendapatan
ORDER BY month 
''').to_dataframe()
rank_revenue_q3

year  month           category    revenue  rank
0  2022      7              Jeans  291556.34     1
1  2022      8              Jeans  317618.18     1
2  2022      9  Outerwear & Coats  307363.79     1

Dari tabel diatas didapatkan informasi :
- Kategori produk dengan pendapatan (status transaksi: complete) terbanyak di bulan Juli 2022 adalah **jeans** dengan total pendapatan sebesar **291.556** USD
- Kategori produk dengan pendapatan (status transaksi: complete) terbanyak di bulan Agustus 2022 adalah **jeans** dengan total pendapatan sebesar **317.618** USD
- Kategori produk dengan pendapatan (status transaksi: complete) terbanyak di bulan September 2022 adalah **Outerwear & Coats** dengan total pendapatan sebesar **307.363** USD

### Number of Products Sold

Bagian ini memperhitungkan jumlah produk yang terjual (dengan status `complete`) setiap bulannya selama Q1 sampai Q3 tahun 2022. Tabel yang digunakan pada query ini adalah tabel `orders`. Di dalam tabel `orders` terdapat kolom `num_of_items` yang menunjukkan jumlah suatu produk. Berikut adalah tabel jumlah produk yang terjual (dengan status `complete`)setiap bulannya selama Q1 sampai Q3 tahun 2022.

In [118]:
# Jumlah produk yang terjual
item_purchased = client.query('''
SELECT DISTINCT
  extract(year FROM created_at) AS year,
  extract(month FROM created_at) AS month,
  SUM(CASE WHEN  status='Complete' THEN num_of_item ELSE 0 END) AS total_item_complete # kolom `num_of_items` table `orders`
FROM bigquery-public-data.thelook_ecommerce.orders
WHERE created_at BETWEEN "2022-01-01" AND "2022-09-30" 
GROUP BY year,month
ORDER BY year,month ASC;
''').to_dataframe()

item_purchased

year  month  total_item_complete
0  2022      1                 1230
1  2022      2                 1179
2  2022      3                 1333
3  2022      4                 1424
4  2022      5                 1508
5  2022      6                 1699
6  2022      7                 1719
7  2022      8                 1871
8  2022      9                 2030

Tabel diatas memperlihatkan jumlah produk yang terjual (dengan status complete) setiap bulannya selama Q1 sampai Q3 tahun 2022. Perhitungan ini digunakan untuk perhitungan GMV.



---



# Business Analysis

## Correlation Analysis

Analisis Korelasi merupakan sebuah analisis yang digunakan untuk mengetahui keeratan hubungan antara beberapa variabel [Link](https://www.dqlab.id/contoh-teknik-analisis-data-korelasi-mengukur-hubungan-2-variabel#:~:text=Analisis%20Korelasi%20merupakan%20sebuah%20analisis,masing%2Dmasing%20variabel%20saling%20mempengaruhi.).  Analisis korelasi yang dilakukan adalah untuk mencari tahu mana yang paling berkaitan dengan jumlah user yang melakukan transaksi dibandingkan jumlah transaksi dan total penjualan. Analisis korelasi dilakukan menggunakan library pandas yaitu fungsi `.corr()` dari dataframe yang merupakan gabungan dari kolom jumlah user, jumlah transaksi dan total penjualan seperti dibawah ini:

In [124]:
# membuat dataframe baru 
corr = pd.DataFrame({'Total_Sales':revenue_all['Total_Sales'],
                    'Complete_Transaction': transaction_status['status_complete'],
                    'Total_Users':num_of_users['num_users_complete']})
corr

Total_Sales  Complete_Transaction  Total_Users
0      71538.0                   855          847
1      67929.0                   815          810
2      77514.0                   929          923
3      83984.0                   975          969
4      89469.0                  1049         1044
5     101666.0                  1160         1150
6     105797.0                  1205         1192
7     113440.0                  1302         1292
8     116349.0                  1366         1354

In [125]:
#analisis korelasi
corr.corr()

Total_Sales  Complete_Transaction  Total_Users
Total_Sales              1.000000              0.997065     0.996838
Complete_Transaction     0.997065              1.000000     0.999955
Total_Users              0.996838              0.999955     1.000000

Dari analisis korelasi diatas dapat disimpulkan : 
- Korelasi antara jumlah user dengan total penjualan sebesar **0.996** atau mendekati 1 yang menunjukkan saling berkaitan satu sama lain.
- Korelasi antara jumlah user dengan jumlah transaksi sebesar **0.999** atau mendekati 1 yang menunjukkan saling berkaitan satu sama lain.

Karena nilai korelasi keduanya sama-sama mendekati 1 maka jumlah user dengan jumlah transaksi dan total penjualan kaitannya sangat kuat.

## Gross Merchandise Value

Perhitungan gross merchandise value mengacu pada nilai barang yang dijual melalui platform Customer-to-Customer(C2C) atau e-commerce [(Investopedia)](https://www.investopedia.com/terms/g/gross-merchandise-value.asp#:~:text=Gross%20merchandise%20value%20(GMV)%20refers,of%20any%20fees%20or%20expenses..). Rumus untuk perhitungan GMV :



```
GMV = Product Sales Price x Number of Products Sold
```



Berikut adalah perhitungan Gross Merchandise Value.

In [119]:
gmv = pd.DataFrame({'Month': ["January", "February", "March", "April", "May", "June", "July", "August", "September"],
                    'Revenues (Complete)': revenue_all['Total_Sales'],
                    'Item Purchased (Complete)': item_purchased['total_item_complete'],
                    'GMV': revenue_all['Total_Sales'] * item_purchased['total_item_complete']})
gmv.round(2)

Month  Revenues (Complete)  Item Purchased (Complete)          GMV
0    January              71538.0                       1230   87991740.0
1   February              67929.0                       1179   80088291.0
2      March              77514.0                       1333  103326162.0
3      April              83984.0                       1424  119593216.0
4        May              89469.0                       1508  134919252.0
5       June             101666.0                       1699  172730534.0
6       July             105797.0                       1719  181865043.0
7     August             113440.0                       1871  212246240.0
8  September             116349.0                       2030  236188470.0

Tabel diatas menunjukkan nilai GMV cenderung mengalami kenaikkan setiap bulannya yang dapat diartikan :
1. Produk yang dijual banyak (quantity)
2. Banyak produk dengan harga mahal yang terjual (price).



---




# Analysis Point

Point penting  yang didapatkan dari analisis yang sudah dilakukan yaitu berdasarkan jumlah transaksi, jumlah user dan total penjualan selalu mengalami kenaikkan setiap bulan selama Q1 sampai Q3 tahun 2022, hal ini menunjukkan bahwa The Look e-commerce berada di kondisi yang baik dari segi bisnis.



---



# Question Point

1. Apakah problem statement yang kamu definisikan di awal dapat terukur ketercapaiannya? berikan pendapatmu!

Jawab: 

Ya dapat tercapai, dari analisis yang sudah dilakukan berdasarkan jumlah transaksi, jumlah user dan total penjualan selalu mengalami kenaikkan setiap bulan selama Q1 sampai Q3 tahun 2022, hal ini menunjukkan bahwa The Look e-commerce berada di kondisi yang baik dari segi bisnis.

2. Berdasarkan hasil analisis yang sudah kamu lakukan dari 6 penjabaran di atas, jika CEO perusahaanmu ingin menargetkan pendapatan di awal kuartal 4 harus mencapai $250000, apakah masuk akal?

Jawab:

Berdasarkan hasil analisis di akhir kuartal 3 total pendapatan mencapai 116.349 USD sehingga target pendapatan di awal kuartal 4 mencapai 250.000 usd tidak tercapai.

3. CEO kamu menargetkan di kuartal 4 ada investor yang dapat menyuntikan dana ke perusahaanmu, dimana investor akan melihat GMV selama 3 kuartal terakhir serta prospek kedepan minimal di kuartal 4 akan seperti apa. Berikan informasi kepada CEO mu berdasarkan trend transaksi, jumlah user yang bertransaksi, dan GMV 3 kuartal terakhir, apakah perusahaanmu layak atau tidak mendapatkan investor baru

Jawab:

Berdasarkan trend transaksi, jumlah user yang bertransaksi, dan GMV 3 kuartal terakhir selalu mengalami kenaikan setiap bulannya. Berdasarkan tren tadi besar kemungkinannya The Look e-commerce akan mengalami tren positif di kuartal 4, sehingga investor baru tidak perlu ragu untuk menginvestasikan dananya ke The Look e-commerce.

---



*NB: angka di tabel sering berubah (T.T)